src/pages/server-sitemap.xml/index.tsx

```diff
@@ -5,23 +5,23 @@
 
 import { getServerSideSitemap } from 'next-sitemap'
 import { GetServerSideProps, GetServerSidePropsContext } from 'next'
+import { allDocsUrls, allTutorialsUrls } from 'lib/sitemap'
 
 export const getServerSideProps: GetServerSideProps = async (
 	ctx: GetServerSidePropsContext
 ) => {
-	const response = await fetch(
-		`https://content.hashicorp.com/api/all-docs-paths`
-	)
+	// returns an array of docs content sitemap objects per slug
+	const docsUrlsPromise = allDocsUrls()
+	// returns an array of tutorials content sitemap objects per slug
+	const tutorialsUrlsPromise = allTutorialsUrls()
+	const sitemapFields = await Promise.all([
+		docsUrlsPromise,
+		tutorialsUrlsPromise,
+	]).catch((error: Error) => {
+		throw new Error('Error generating server-sitemap.xml', error)
+	})
```

Comment:
This code currently errors out, I think its because the return arrays need to be flattened one more time. 

<img width="1128" alt="Screenshot 2023-03-07 at 6 02 35 PM" src="https://user-images.githubusercontent.com/36613477/223600680-2f14d2b6-2ceb-4273-9460-19fdfdb9b66b.png">


Here's some suggested pseudocode, i'm not sure we need the concurrency with `promise.all` since its just two calls, but curious to hear your thoughts there! 

```suggestion
		const docsUrls = await allDocsUrls()
	// returns an array of tutorials content sitemap objects per slug
	const tutorialsUrls = await allTutorialsUrls()
	const sitemapFields = [...docsUrls, ...tutorialsUrls]
```

Reply:
I agree, I ran these concurrently when we were also generating Integrations sitemap URLs, but we have omitted them recently. So the concurrency is an artifact of a previous effort.